core/conn/jdbc_type2/native/ResStatisticsStatement.cpp

```diff
@@ -1537,39 +1537,31 @@ void ResStatisticsStatement::setStatistics(SRVR_STMT_HDL *pSrvrStmt, SQLSTATS_TY
                 initSqlStatsItems(measStatsItems_, MAX_MEASSTATS_ENTRY, &sqlStatsDesc_[currStatsDescEntry_]);
                 measStatsItems_[0].statsItem_id = SQLSTATS_ACT_ROWS_ACCESSED;
                 measStatsItems_[1].statsItem_id = SQLSTATS_ACT_ROWS_USED;
-                measStatsItems_[2].statsItem_id = SQLSTATS_MSG_COUNT;
-                measStatsItems_[3].statsItem_id = SQLSTATS_MSG_BYTES;
-                measStatsItems_[4].statsItem_id = SQLSTATS_STATS_BYTES;
-                measStatsItems_[5].statsItem_id = SQLSTATS_DISK_IOS;
-                measStatsItems_[6].statsItem_id = SQLSTATS_LOCK_WAITS;
-                measStatsItems_[7].statsItem_id = SQLSTATS_LOCK_ESCALATIONS;
-                measStatsItems_[8].statsItem_id = SQLSTATS_DP2_CPU_BUSY_TIME;
-                measStatsItems_[9].statsItem_id = SQLSTATS_SQL_CPU_BUSY_TIME;
-                measStatsItems_[10].statsItem_id = SQLSTATS_SQL_SPACE_ALLOC;
-                measStatsItems_[11].statsItem_id = SQLSTATS_SQL_SPACE_USED;
-                measStatsItems_[12].statsItem_id = SQLSTATS_SQL_HEAP_ALLOC;
-                measStatsItems_[13].statsItem_id = SQLSTATS_SQL_HEAP_USED;
-                measStatsItems_[14].statsItem_id = SQLSTATS_EID_SPACE_ALLOC;
-                measStatsItems_[15].statsItem_id = SQLSTATS_EID_SPACE_USED;
-                measStatsItems_[16].statsItem_id = SQLSTATS_EID_HEAP_ALLOC;
-                measStatsItems_[17].statsItem_id = SQLSTATS_EID_HEAP_USED;
-                measStatsItems_[18].statsItem_id = SQLSTATS_OPENS;
-                measStatsItems_[19].statsItem_id = SQLSTATS_OPEN_TIME;
-                measStatsItems_[20].statsItem_id = SQLSTATS_PROCESS_CREATED;
-                measStatsItems_[21].statsItem_id = SQLSTATS_PROCESS_CREATE_TIME;
-                measStatsItems_[22].statsItem_id = SQLSTATS_REQ_MSG_CNT;
-                measStatsItems_[23].statsItem_id = SQLSTATS_REQ_MSG_BYTES;
-                measStatsItems_[24].statsItem_id = SQLSTATS_REPLY_MSG_CNT;
-                measStatsItems_[25].statsItem_id = SQLSTATS_REPLY_MSG_BYTES;
-                measStatsItems_[26].statsItem_id = SQLSTATS_SCRATCH_FILE_COUNT;
-                measStatsItems_[27].statsItem_id = SQLSTATS_SCRATCH_BUFFER_BLOCK_SIZE;
-                measStatsItems_[28].statsItem_id = SQLSTATS_SCRATCH_BUFFER_BLOCKS_READ;
-                measStatsItems_[29].statsItem_id = SQLSTATS_SCRATCH_BUFFER_BLOCKS_WRITTEN;
-                measStatsItems_[30].statsItem_id = SQLSTATS_SCRATCH_OVERFLOW_MODE;
-                measStatsItems_[31].statsItem_id = SQLSTATS_SCRATCH_READ_COUNT;
-                measStatsItems_[32].statsItem_id = SQLSTATS_SCRATCH_WRITE_COUNT;
-                measStatsItems_[33].statsItem_id = SQLSTATS_UDR_CPU_BUSY_TIME;
-                // MAX_MEASSTATS_ENTRY is set to 34
+                measStatsItems_[2].statsItem_id = SQLSTATS_SE_IOS;
```

Comment:
Looks like you took all the old DP2-specific counters out and added some "SE" counters. Is my understanding correct?